bittensor/core/extrinsics/transfer.py

```diff
@@ -161,17 +158,6 @@ def transfer_extrinsic(
         logging.info(f"\t\tFor fee: \t<blue>{fee}</blue>")
         return False
 
-    # Ask before moving on.
-    if prompt:
-        if not Confirm.ask(
-            "Do you want to transfer:[bold white]\n"
-            f"  amount: {transfer_balance}\n"
-            f"  from: {wallet.name}:{wallet.coldkey.ss58_address}\n"
-            f"  to: {dest}\n"
-            f"  for fee: {fee}[/bold white]"
-        ):
-            return False
-
```

Comment:
Same point here, maybe we can just log all this info

Reply:
done